### PR TITLE
Feature/unit test coverage

### PR DESCRIPTION
**Pull request by david-kasali**

### CODE REVIEW

**Comment by paul-burnett** [comment]
Revert package.json changes

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "register_a_food_business_front_end",
-  "version": "10.0.1",
+  "version": "9.5.0",
```
Comment (paul-burnett) [inline package.json:3]: Revert these changes

File: pages_unit_tests/establishment-address-select.test.js [hunk 2]
```diff
@@ -3,8 +3,11 @@ import { mount, shallow } from "enzyme";
 
 const testCumulativeAnswers = {
   example: "test answer"
+ // editModeFirstPage: "registration-role"
```
Comment (paul-burnett) [inline pages_unit_tests/establishment-address-select.test.js:6]: Remove comment

File: src/server/routes.test.js [hunk 3]
```diff
@@ -37,6 +37,12 @@ describe("Function: routes", () => {
   afterEach(() => {
     jest.clearAllMocks();
   });
+  
+  it("Should call get", () => {
+    process.env.NODE_ENV = "production"
```
Comment (paul-burnett) [inline src/server/routes.test.js:42]: Not sure I like this, shouldn't be using production in a test

File: src/server/services/data-transform.service.js [hunk 4]
```diff
@@ -4,14 +4,6 @@
  */
 
 const { logEmitter } = require("./logging.service");
```
Comment (paul-burnett) [inline src/server/services/data-transform.service.js:6]: This looks like it's undoing the changes from release 10.0.0??

Comment (jj-perry) [reply src/server/services/data-transform.service.js]: Think these have been reversed. No changes to data-transform listed now.

File: src/server/routes.test.js (hunk 3 above)
Comment (jj-perry) [reply src/server/routes.test.js]: Removed this test again.

File: pages_unit_tests/establishment-address-select.test.js (hunk 2 above)
Comment (jj-perry) [reply pages_unit_tests/establishment-address-select.test.js]: Again no changes to this file anymore so believe this has already been done.

File: package.json (hunk 1 above)
Comment (jj-perry) [reply package.json]: Reverted these changes and upgraded packages that are causing snyk failures. Snyk only seems to report on master branch however so won't reflect the updates in the checks yet.

**Review by paul-burnett — APPROVED** [review]
